drivers/char/rust_example_3.rs

```diff
@@ -36,6 +36,14 @@ impl KernelModule for RustExample3 {
         println!("[3] Parameters:");
         println!("[3]   my_bool:  {}", my_bool.read());
         println!("[3]   my_i32:   {}", my_i32.read());
+
+        // Including this large variable on the stack will trigger a call to 
```

Comment:
Nice! The 2/3/4 examples are intended to be copy-pastes of the first one (for the CI). Can you please put this into all of them?

In the future we will probably move these runtime tests somewhere else, but for now it is good to have them here. +1

Reply:
done